omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -359,6 +359,10 @@
 # recommend false; set to true as default for backward compatibility
 default['supermarket']['owners_can_remove_artifacts'] = true
 
+# The below flag is to allow arbitrary host headers. The scenarios
+# in which this flag needs to be set to false is e.g. if supermarket
+# runs behind an AWS ELB load balancer.
+default['supermarket']['disable_host_header_attack'] = true
```

Comment:
is this attribute used somewhere in the cookbook to disable host header attack @RajeshPaul38 ?

Reply:
No, not in cookbook. We are using it in the application initializer.

Reply:
Please mention in the documention for this flag that we need to make sure value of `ALLOWED_HOST` be present for making sure which domain to redirect the requests to .. or something like that

Reply:
@msys-sgarg actually it's derived from FQDN. But in case the FQDN is different from supermarket domain name only then we need to explicitly specify it in the attributes.
Also I would need to mention the usage of the new flag: `disable_host_header_attack` in the documentation. Thanks for pointing it out. Will make necessary changes.

Reply:
@msys-sgarg I've made the documentation changes. Pls review.

Reply:
documentation looks good